apps/dashboard/src/core-ui/batch-upload/upload-step.tsx

```diff
@@ -20,117 +21,111 @@ export const UploadStep: React.FC<UploadStepProps> = ({
   getInputProps,
 }) => {
   return (
-    <Flex flexGrow={1} align="center" overflow="auto">
-      <Container maxW="container.page">
-        <Flex gap={8} flexDir={{ base: "column", md: "row" }}>
-          <div className="relative aspect-square w-full md:w-1/2">
-            <div
+    <div className="flex flex-col items-center gap-8 md:flex-row">
+      <div className="relative aspect-square w-full md:w-1/2">
+        <div
+          className={cn(
+            "flex h-full cursor-pointer rounded-md border border-border hover:border-blue-500",
+            hasFailed ? "bg-red-200" : "bg-card",
```

Comment:
there's no longer any container here, is that not required?

Reply:
Same reason with comment above. I'm making change to this design hopefully you'd like it